examples/superbible/tesselation_with_geometry/simple_teselation_with_geomatry.gpr

```diff
@@ -0,0 +1,15 @@
+limited with "../../../opengl-glfw.gpr";
+
+project Simple_Teselation_With_Geomatry is
```

Comment:
This could use fixes to spelling, should be `Simple_Tessellation_With_Geometry`. Please also fix all other misspellings of `tessellation` (mind the double `l`).